examples/sensors/main.c

```diff
@@ -42,23 +42,23 @@ static void timer_fired(__attribute__ ((unused)) int arg0,
   if (humidity)     humidity_read_sync(&humi);
   if (ninedof)      ninedof_read_acceleration_sync(&ninedof_x, &ninedof_y, &ninedof_z);
 
-  if (isl29035)       printf("ISL29035:   Light Intensity: %d\n", light);
-  if (tmp006)         printf("TMP006:     Temperature:     %d\n", tmp006_temp);
-  if (tsl2561)        printf("TSL2561:    Light:           %d lux\n", tsl2561_lux);
-  if (lps25hb)        printf("LPS25HB:    Pressure:        %d\n", lps25hb_pressure);
-  if (temp)           printf("Temperature:                 %d deg C\n", temp/100);
-  if (humi)           printf("Humidity:                    %u%%\n", humi/100);
-  if (ninedof)        printf("FXOS8700CQ: X:               %d\n", ninedof_x);
-  if (ninedof)        printf("FXOS8700CQ: Y:               %d\n", ninedof_y);
-  if (ninedof)        printf("FXOS8700CQ: Z:               %d\n", ninedof_z);
+  if (isl29035)       printf("ISL29035:   Light Intensity: %d\r\n", light);
+  if (tmp006)         printf("TMP006:     Temperature:     %d\r\n", tmp006_temp);
+  if (tsl2561)        printf("TSL2561:    Light:           %d lux\r\n", tsl2561_lux);
+  if (lps25hb)        printf("LPS25HB:    Pressure:        %d\r\n", lps25hb_pressure);
+  if (temp)           printf("Temperature:                 %d deg C\r\n", temp/100);
+  if (humi)           printf("Humidity:                    %u%%\r\n", humi/100);
+  if (ninedof)        printf("FXOS8700CQ: X:               %d\r\n", ninedof_x);
+  if (ninedof)        printf("FXOS8700CQ: Y:               %d\r\n", ninedof_y);
+  if (ninedof)        printf("FXOS8700CQ: Z:               %d\r\n", ninedof_z);
   /* *INDENT-ON* */
 
-  printf("\n");
+  printf("\r\n");
```

Comment:
Can you pull the `\r` changes out of this PR? I don't think they have anything to do with this driver.

I understand some serial terminals require CRLF line endings, and maybe all the tock examples could use them, but right now we pretty consistently use just `\n` endings (excepting one BLE example, apparently)

Reply:
My shell was one of the ones that needs the \r\n. I'll delete it.